app/javascript/modules/listingDetailsLottery/ListingDetailsLotterySearchFooter.tsx

```diff
@@ -16,7 +17,11 @@ export const ListingDetailsLotterySearchFooter = ({
   return (
     <>
       <div className="bg-gray-100 py-4 text-center">
-        <a href="http://sfmohcd.org/housing-preference-programs" target="_blank">
+        <a
+          className="lottery-results-modal-anchor"
```

Comment:
do we not have a tailwind utility that approximates this color? not sure we've adopted a team-wide preference, but my own would be for trying tailwind before stylesheets. or maybe this case is more complicated, as this modal proves to be over and over again 🤪

Reply:
Good idea! I found a close color and used a tailwind class in the jsx instead of the sass file. Also replaced another hard coded hex value with a sass var. Thanks @jacksongonzales!

Reply:
👏